assets/i18n/en.json

```diff
@@ -69,5 +69,6 @@
   "Day": "Day",
   "Month": "Month",
   "Year": "year",
-  "Close": "Close"
+  "Close": "Close",
+  "OPENS_IN_SAME_TAB" : "(opens in same tab)"
```

Comment:
I think the text here should be "Opens in a new tab" also applicable for English. Please check with the Pega dev on this, since the link is expected to open in a new tab